scripts/testing/setup-mock-test-env.py

```diff
@@ -343,13 +383,15 @@ def setup_mock(mock_command):
 
     if ns.init:
         setup_mock(mock_cmd)
-        mock_init_run = True
-        if ns.install:
-            install_packages_to_mock(mock_cmd, ns.install)
+        if ns.install_pip is None:
+            install_required_pip_packages(mock_cmd)
 
-    if ns.install and not mock_init_run:
+    if ns.install:
         install_packages_to_mock(mock_cmd, ns.install)
 
+    if ns.install_pip is not None:
```

Comment:
Does it makes sense to call pip if `ns.install_pip` is an empty string?

Reply:
Yes if you want to have the behavior in the help, no otherwise.